### PR TITLE
Fix compiling warnings

### PR DESCRIPTION
**Pull request by cam-parra**
Signed-off-by: Cam Parra <caeparra@gmail.com>

### CODE REVIEW

File: libzmix/src/zkl/mod.rs [hunk 1]
```diff
@@ -3,7 +3,7 @@ pub mod spec;
 pub struct Parser {}
 
 impl Parser {
-    pub fn parse(proof_spec: &spec::ProofSpec, witness: &Witness) -> Result<Proof, ZKLError> {
+    pub fn parse(_proof_spec: &spec::ProofSpec, _witness: &Witness) -> Result<Proof, ZKLError> {
```
Comment (lovesh) [inline libzmix/src/zkl/mod.rs]: Why not use `#[allow(unused_variables)]`. The reason compiler is complaining about the name is that the function is not implemented. What if we had a Parser trait and this was just one of the implementation. We would not want to change arg names.

File: libzmix/src/zkl/mod.rs [hunk 2]
```diff
@@ -21,7 +21,7 @@ impl WitnessBuilder {
 pub struct Proof {}
 
 impl Proof {
-    pub fn verify(&self, proof_spec: &spec::ProofSpec) -> Result<bool, ZKLError> {
+    pub fn verify(&self, _proof_spec: &spec::ProofSpec) -> Result<bool, ZKLError> {
```
Comment (lovesh) [inline libzmix/src/zkl/mod.rs]: Same comment as above

File: libzmix/src/zkl/mod.rs (hunk 1 above)
Comment (cam-parra) [reply libzmix/src/zkl/mod.rs]: I agree with this. Making the changes.

File: libzmix/src/zkl/mod.rs [hunk 3]
```diff
@@ -2,8 +2,9 @@ pub mod spec;
 
 pub struct Parser {}
 
+#[allow(unused_variables)]
 impl Parser {
-    pub fn parse(_proof_spec: &spec::ProofSpec, _witness: &Witness) -> Result<Proof, ZKLError> {
+    pub fn parse(proof_spec: &spec::ProofSpec, _witness: &Witness) -> Result<Proof, ZKLError> {
```
Comment (lovesh) [inline libzmix/src/zkl/mod.rs]: What about `_witness`?

**Pull request merged**